sdk/src/main/java/com/horizen/transaction/RegularTransaction.java

```diff
@@ -26,7 +26,7 @@
     extends SidechainTransaction<Proposition, NoncedBox<Proposition>>
 {
     private List<RegularBox> inputs;
-    private List<BoxData> outputs;
+    private List<BoxData<? extends Proposition, ? extends NoncedBox<? extends Proposition>>> outputs;
```

Comment:
`? extends` just for minimization of compilation errors. `RegularTransaction` class will be removed soon.